src/module-1/card/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+    <title>Product Cart</title>
+</head>
+<body>
+    <div class="product-card">
+        <div class="product-card__inner">
+            <div class="product-card__image">
+                <img src="images/apple-watch.png">
+            </div>
+            <div class="product-card__wrapper">
+                <div class="product-card__rating rating">
+                    <span class="rating__count">3.4</span>
+                    <img class="rating__image" src="./images/star.svg">
+                </div>
+                <div class="rating__price">
+                    $399
+                </div>
+            </div>
+            <h3 class="product-card__name">
+                Apple Watch Series 4 GPS
+            </h3>
+            <div class="product-card__description">
+                Redesigned from scratch and completely revised.
+            </div>
+        </div>
+            <div class="product-card__buttons">
+                <button class="button-wishlist">
+                    <img class="button-wishlist__image" src="images/heart.svg">
```

Comment:
why not to use :before pseudo selector and content: url( path to img) on span with text ?